node-require/calculate.js

```diff
@@ -0,0 +1,20 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
```

Comment:
It would be best to create variables for your user input here. It will save you some typing and cleanup your code. For example:

```JAVASCRIPT
const num1 = Number(process.argv[2]);
const op = process.argv[3];
const num2 = Number(process.argv[4]);
```